.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.0-dev
+current_version = 0.1.0
```

Comment:
We should adjust `__init__.py` instead, since we still want this to be -dev.

Reply:
Looks like it already [is](https://github.com/IBM/sqlalchemy-ibmi/blob/master/sqlalchemy_ibmi/__init__.py#L22), so this change can just be removed.